Dockerfile

```diff
@@ -0,0 +1,73 @@
+# Python 3.8
+FROM python:3.9
+
+EXPOSE 8501
+
+# SHELL ["/bin/bash", "--login", "-c"]
+
+LABEL version="0.1.0"
+
+LABEL description="An automated text-mining tool written in Python to measure \
+    the technical responsibility of students in computer science courses, being \
+    used to analyze students' markdown reflection documents and five questions \
+    survey based on Natural Language Processing in the Department of Computer \
+    Science at Allegheny College."
+
+LABEL maintainer="Group 5"
```

Comment:
We definitely appreciate your work on this and give you credit for it. I think that this should be more specific than `Group 5`. Perhaps you can include the Github usernames of contributors to this Docker image